.circleci/build_win.ps1

```diff
@@ -1,4 +1,11 @@
 cd "$PSScriptRoot\.."
+
+Write-Host "Force release: $env:FORCE_RELEASE / $Env:FORCE_RELEASE"
+
+if ("$Env:FORCE_RELEASE" -eq "ON") {
+	Out-File -FilePath prerelease.txt
+}
+
```

Comment:
Just
```suggestion

if ("$Env:FORCE_RELEASE") { Out-File -FilePath prerelease.txt }

```
should work I think.

Reply:
Jep, copied your fix in the end.

Reply:
Rebased the testing PR onto this again - that should print the version in the test run to make sure this worked.